loadtest/control/simplecontroller/controller.go

```diff
@@ -64,7 +64,11 @@ func (c *SimpleController) Run() {
 			waitAfter: 1000,
 		},
 		{
-			run:       c.createPost,
+			run:       c.addReaction,
```

Comment:
Shouldn't this action be performed after a post is created? even though other user entities are creating posts

Reply:
Not really. We are usually not trying to react to our own post. Also the order of these actions might change soon. So unless one action is strictly required to run before another (like with login) I wouldn't worry too much.